src/GregClient/Utility/Utilities.cs

```diff
@@ -9,6 +9,24 @@
 
 namespace Greg.Utility
 {
+    public static class AppSettingMgr
+    {
+        public static KeyValueConfigurationElement GetItem(String key)
+        {
+            try
+            {
+                var dllPath = new Uri(Assembly.GetExecutingAssembly().GetName().CodeBase).LocalPath;
```

Comment:
what is this set to when running in DynamoRevit - is it different when using a standard install?

Reply:
I do not see why they would be different ....

Reply:
Isn't this project common for standard and DynamoRevit ?

Reply:
well, in a standard Dynamo install the executing assembly should be under Revit/Addins/DynamoForRevit - 
and running locally this may be the dynamo core folder - 
I guess it does not matter as long as the config is present?

Reply:
Yep..that should be the case...